packages/web-components/src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -46,6 +63,16 @@ export class VaOnThisPage {
     });
   };
 
+  connectedCallback() {
+    i18next.on('languageChanged', () => {
+      forceUpdate(this.el);
```

Comment:
This is to support dynamic translation and is copied from the same pattern in `<va-text-input>`.

https://github.com/department-of-veterans-affairs/component-library/blob/d5b47709b42774827462383604f403667a9a6937/packages/web-components/src/components/va-text-input/va-text-input.tsx#L186-L194